modular_skyrat/modules/colony_fabricator/code/design_datums/tools.dm

```diff
@@ -0,0 +1,194 @@
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 1.75,
+		/datum/material/titanium = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_ENGINEERING_ADVANCED,
+	)
+
+// Welder that takes no fuel or power to run but is quite slow, at least it sounds cool as hell
+
+/datum/design/colony_arc_welder
+	name = "Arc Welder"
+	id = "colony_arc_welder"
+	build_type = COLONY_FABRICATOR
+	build_path = /obj/item/weldingtool/electric/arc_welder
+	materials = list(
+		/datum/material/iron = HALF_SHEET_MATERIAL_AMOUNT,
+		/datum/material/glass = SMALL_MATERIAL_AMOUNT * 5,
+		/datum/material/plasma = HALF_SHEET_MATERIAL_AMOUNT * 1.5,
+	)
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_ENGINEERING_ADVANCED,
+	)
+
+// Slightly slower drill that fits in backpacks
+
+/datum/design/colony_compact_drill
+	name = "Compact Mining Drill"
+	id = "colony_compact_drill"
+	build_type = COLONY_FABRICATOR
+	build_path = /obj/item/pickaxe/drill/compact
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 3,
+		/datum/material/glass = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_MINING,
+	)
+
+// Various designs that get added to the colony fab
+
+/datum/design/holosignatmos/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/analyzer/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/extinguisher/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/cable_coil/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/airlock_painter/decal/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/airlock_painter/decal/tile/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/holosignengi/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/inducer/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/multitool/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/tscanner/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/pipe_painter/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/rwd/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/bolter_wrench/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/rpd/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/rpd_upgrade/unwrench/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/rtd_loaded/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/rcd_ammo/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/light_replacer/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/rld_mini/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/miningsatchel_holding/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
```

Comment:
This stuff is still dependent on research, yeah?

Reply:
No, the point of the fabricator is that it comes with extra designs regardless of research, however it doesn't get any new designs through research (I was thinking of adding buyable upgrades for it later that give it other unique items). You also can't upgrade the fabricator to get any of the benefits of that either + the fact you have to buy it makes me feel like these aren't a concern.

Reply:
Hmmm, okay, I'll let it happen for now, then.

If it becomes _the_ way to obtain mining satchels of holding, however, I think we'll need to remove this from it, I'm not sure it would be too healthy for the round progression otherwise (as nobody would research that node ever, pretty much)